Fix make check target

**How to categorize this issue?**
<!--
Please select area, kind, and priority for this issue. This helps the community categorizing it.
Replace below TODOs or exchange the existing identifiers with those that fit best in your opinion.
If multiple identifiers make sense you can also state the commands multiple times, e.g.
  /area control-plane
  /area auto-scaling
  ...

"/area" identifiers:     audit-logging|auto-scaling|backup|certification|control-plane-migration|control-plane|cost|delivery|dev-productivity|disaster-recovery|documentation|high-availability|logging|metering|monitoring|networking|open-source|ops-productivity|os|performance|quality|robustness|scalability|security|storage|testing|usability|user-management
"/kind" identifiers:     api-change|bug|cleanup|discussion|enhancement|epic|impediment|poc|post-mortem|question|regression|task|technical-debt|test
"/priority" identifiers: 1|2|3|4|5 (ordered from greatest to least)
-->
/area control-plane
/area dev-productivity
/kind bug
/priority 3

**What happened**:
`make check` target fails on the CI/CD pipeline with the following error:
```
E0428 07:04:53.646057   15994 main.go:41] Error: Failed making a parser: unable to add directory "github.com/gardener/dependency-watchdog/api/...": no loaded build package for github.com/gardener/dependency-watchdog/api
make: *** [Makefile:55: check] Error 1
```

**What you expected to happen**:
`make check` should succeed.

--- a/hack/check-imports.sh
+++ b/hack/check-imports.sh
@@ -30,7 +30,7 @@ set -o pipefail
 echo "> Check Imports"
 
 this_module=$(go list -m)
-export GO111MODULE=on
+export GO111MODULE=off
 
 packages=()
 for p in "$@" ; do

--- a/hack/check.sh
+++ b/hack/check.sh
@@ -30,7 +30,7 @@ done
 echo "> Check"
 
 echo "Executing golangci-lint"
-golangci-lint -v run $GOLANGCI_LINT_CONFIG_FILE --timeout 10m $@
+golangci-lint run $GOLANGCI_LINT_CONFIG_FILE --timeout 10m $@
 
 
 echo "Executing gomegacheck"

